packages/docs/PropTables/DropdownPropTable.tsx

```diff
@@ -33,10 +33,10 @@ const dropdownProps: Prop[] = [
   },
   {
     name: 'options',
-    types: 'Array<DropdownItem | DropdownLinkItem>',
+    types: 'Array<Item | LinkItem>',
     description: (
       <>
-        Accepts an array of <Code>DropdownItems</Code> and <Code>DropdownLinkItems</Code>. See example for usage.
+        Accepts an array of <Code>Items</Code> and <Code>LinkItems</Code>. See example for usage.
```

Comment:
🍹 Should this be `Array of Items _or_ LinkItems` ?

Reply:
Good observation. The Dropdown actually accepts a mix of both.